### PR TITLE
fix: Handled crash while only a single point is being fetched in Logic Analyzer

### LINKED ISSUE #1139: App crashes when only a single data point is fetched in Logic Analyzer
**Actual Behaviour**

Index 1 is used although the size of the array of the fetched data point is 1 and so the app crashes

**Expected Behaviour**

The app shouldn't crash

**Steps to reproduce it**

Go to Logic Analyzer -> Analyze until only a single data point is fetched

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by harsh-2711**
Fixes #1139 

**Changes**: Handled the case of a single data point being fetched using if..else statement

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/pslab-android/files/2138586/app-debug.zip)

### CODE REVIEW

**Comment by harsh-2711** [comment]
@CloudyPadmal @abhinavraj23 @Vikum94 @Avjeet Please review. It's a small PR.

**Comment by CloudyPadmal** [comment]
@Vikum94 @abhinavraj23 @Avjeet please install the APK and test for functionality. Then update here if it's fixed.

File: app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java [hunk 1]
```diff
@@ -426,22 +426,26 @@ private void singleChannelEveryEdge(double[] xData, double[] yData) {
...
+                    tempInput.add(new Entry(xaxis.get(0), yaxis.get(0)));
+                    tempInput.add(new Entry(xaxis.get(0), yaxis.get(1)));
                 }
+                for (int i = 1; i < xaxis.size() - 1; i++) {
+                    if (yaxis.get(i).equals(yaxis.get(i + 1)))
+                        tempInput.add(new Entry(xaxis.get(i), yaxis.get(i)));
+                    else {
+                        tempInput.add(new Entry(xaxis.get(i), yaxis.get(i)));
+                        tempInput.add(new Entry(xaxis.get(i), yaxis.get(i + 1)));
+                    }
```
Comment (abhinavraj23) [inline app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java]: @harsh-2711 Please remove this extra space

**Comment by harsh-2711** [comment]
@abhinavraj23 Changes done.

**Pull request merged**